import_pg_dump.sh

```diff
@@ -16,7 +17,22 @@ if [[ ! -f "$DUMPFILE" ]]; then
     exit 1
 fi
 
-dropdb dryad_repo
-createdb -U dryad_app -E UNICODE dryad_repo
-pg_restore -j $THREADS -d dryad_repo -U dryad_app $DUMPFILE 
+if [[ "$DBHOST" == "" ]]; then
+	DBHOST=$DRYAD_DB_HOST
+    echo "Exiting."
+    exit 1;
+fi
 
+if [[ "$DBHOST" == "" ]]; then
+	psql -U dryad_app -d dryad_repo -c "select pg_terminate_backend(pid) from pg_stat_activity where datname='dryad_repo'"
+	dropdb dryad_repo
+	createdb -U dryad_app -E UNICODE dryad_repo
+	pg_restore -j $THREADS -d dryad_repo -U dryad_app $DUMPFILE 
+fi
+
+if [[ "$DBHOST" != "" ]]; then
```

Comment:
The code within this "if" asks for a database password twice... it doesn't seem to be using the .pgpass file, but it works fine if env var PGPASSWORD is set. Should the password be a required parameter of the script?

Reply:
I'm making edits to this in #38.